index.php

```diff
@@ -1,13 +1,56 @@
 <?php
 
+require __DIR__ . '/vendor/autoload.php';
+
 if ($_SERVER['REQUEST_METHOD'] === 'POST') {
-     // The request is using the GET method
   $state = $_POST['state'];
-  echo get_hospitals($state);
+  $good_or_bad = $_POST['good_or_bad'];
+  echo render_hos(get_hos($state, $good_or_bad));
```

Comment:
Should probably break these things out into classes. Better yet, try the command/handler pattern.